src/components/encounter/ohri-encounter-form.component.tsx

```diff
@@ -125,6 +125,9 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
           flattenedFieldsTemp.push(question);
           if (question.type == 'obsGroup') {
             question.questions.forEach(groupedField => {
+          if (groupedField.questionOptions.rendering == 'fixed-value' && !groupedField['fixedValue']) {
+            groupedField['fixedValue'] = groupedField.value;
+          }
```

Comment:
Please fix the indentation so we can get this in

Reply:
Done